src/features/payment/components/PaymentPspDrawer/PaymentPspDrawer.tsx

```diff
@@ -125,8 +125,11 @@ const sortBy =
     const fieldA = a[field];
     const fieldB = b[field];
     const order = direction === "asc" ? 1 : -1;
-
-    return fieldA && fieldB ? (fieldA > fieldB ? order : -order) : -order;
+    return fieldA !== undefined && fieldB !== undefined
```

Comment:
With strict equality you miss "null" check. In this case is better to use simple one (`!=`) or add also "null" check

Reply:
this was omitted as the related API 
`/payment-methods/{id}/fees` is granted by open API to return an object 
CalculateFeeResponse containing an array of Bundle who has the sorted attribute **taxPayerFee**.

The value **null** is granted to never happen as per OpenAPI signature. The undefined control is added only to avoid crashes in case the attribute is missing due to misconstructed payloads.

Reply:
Ok if null value never happen undefined check is ok